CommandLine/protos/resources/Font.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto2";
+package buffers.resources;
+
+import "options.proto";
+
+message Font {
+  message Glyph {
+    optional uint32 character = 1; 
+    optional uint32 x = 2;
+    optional uint32 y = 3; 
+    optional uint32 width = 4 [(gmx) = "w"];
+    optional uint32 height = 5 [(gmx) = "h"]; 
+    optional int32 shift = 6; 
+    optional int32 offset = 7;
+  }
+
+  message Range {
+    optional int32 min = 1 [(gmx) = "GMX_SPLIT/0"];
+    optional int32 max = 2 [(gmx) = "GMX_SPLIT/1"];
+  }
+
+  optional string name = 1;
+  optional int32 id = 2 [(gmx) = "GMX_DEPRECATED"];
+
+  optional string font_name = 3 [(gmx) = "name"];
+  optional int32 size = 4;
+  optional bool bold = 5;
+  optional bool italic = 6;
+
+  repeated Range ranges = 7 [(gmx) = "ranges/range0"];
+  repeated Glyph glyphs = 8 [(gmx) = "glyphs/glyph"];
+  
+  optional bool renderhq = 10;
+  optional uint32 charset = 11;
+  optional uint32 aa = 12;

```

Comment:
I'd name this "antialias," I believe.

Reply:
Well that's what it is, but why? Then we have to add the gmx custom option, which is ok, it makes our format more human-readable. I'll change it.